### ISSUE TITLE
Splitting the scanning work into multiple drives

### ISSUE THREAD

**Issue by wowkkwow** [0x90d/videoduplicatefinder#197]
**Environment**
 - OS: windows 10 pro 21H1 19043.1165

**Describe the solution you'd like**
I have used Rygen 5800X(8core 16thread), now use Rygen 5900x(12core 24thread).

Both good for VDF Max Degree of parallelism option.

I have many HDDs, drive mounted Google drive and other network drives.

So it takes 3~14 hours or more time to scan a whole duplicated files.

Unless the network mounted drives, the Max Degree of Paralleism Option works good in value of 4~5.

But with the network mounted drives or the higher value of 6, the total working time takes longer.

So I monitored windows resource manger
Value of 4 paralleism optioned work uses 90% of only one HDD (HDD has speed 60MB/sec ~100MB). 

With network mounted drive it takes twice or more time.
I used value of 2 with network mounted drive scanning work.

So It would be good for multiple drives if the scanning works were split into each drives.
It would low the load of a single drive and could use more value of option for mutil-core cpu.

**Comment by 0x90d** [comment]
VDF doesn't know if a volume is a different hard disk or if its mounted and if mounted whether its local network or a cloud drive. The only possible solution is to let the user define a value for each entry in the include list.

**Comment by wowkkwow** [comment]
In case of software just like the WinCatalog, it recognizes drive as the same with HDD's serial number, after user changed drive letter and path.

The network drive and folder has serial number, too.

I hope this info would be help for the solution.

**Comment by 0x90d** [comment]
I have no plans for HDD detection. This is out of scope for a simple program like this. Even splitting them is too specific while on the other hand requires major changes to the scan part. Most of the time spent is only when it launches ffmpeg/ffprobe processes to obtain media info and gray bytes. This happens only once because these information are saved.
